src/app/utils/chart-utils.ts

```diff
@@ -69,3 +69,38 @@ export const filterHourlyActiveAccounts = (
 ): ActiveAccounts[] | undefined => {
   return windows?.filter((value, index) => index % 12 === 0)
 }
+
+export const sumBucketsByChucks = <T>(
```

Comment:
looking at all loops here this logic is so custom that I don't think we need sumKey dataKey args and generics, but Ok.

Reply:
Was just thinking that it might be useful to reuse, for all time chart, or something similar.